Extension/src/LanguageServer/configurations.ts

```diff
@@ -1315,7 +1315,7 @@ export class CppProperties {
                         continue;
                     }
                     // Squiggle when the compiler's path has spaces without quotes but args are used.
-                    compilerPathNeedsQuotes = compilerPathAndArgs.additionalArgs
+                    compilerPathNeedsQuotes = (compilerPathAndArgs.additionalArgs && compilerPathAndArgs.additionalArgs.length > 0)
```

Comment:
This same type of error has occurred before. It would be nice if there were a way to track down any other cases.

Reply:
Can you also add the "compilerPathAndArgs.additionalArgs.length >0" check after line 1039?

Reply:
> Can you also add the "compilerPathAndArgs.additionalArgs.length >0" check after line 1039?

Fixed.